### ISSUE TITLE
Different file naming for scoped request test generator

### ISSUE THREAD

**Issue by hegwin** [rspec/rspec-rails#2074]
### Subject of the issue

When I execute a `rails g` command to generate a request test, it generates a file whose name contains nested scope as its prefix. It is different from other spec generator, eg. controller test or model test.

### Your environment
* Ruby version: 2.5.3
* rspec-core version: 3.8.0

### Steps to reproduce

When I run `$ rails g rspec:request api/v1/locations`

I got 
```
Running via Spring preloader in process 47423
      create  spec/requests/api/v1/api_v1_locations_spec.rb
                                   ^^^^^^^^
```

When I run `$ rails g rspec:controller api/v1/locations`

I got
```
Running via Spring preloader in process 48480
      create  spec/controllers/api/v1/locations_controller_spec.rb
```

When I run `$ rails g rspec:model a/b/c`

I got

```
Running via Spring preloader in process 57281
      create  spec/models/a/b/c_spec.rb
      invoke  factory_bot
      create    spec/factories/a_b_cs.rb
```

### Expected behavior
`$ rails g rspec:request api/v1/locations` should generate a file `spec/requests/api/v1/locations_spec.rb`

### Actual behavior
`$ rails g rspec:request api/v1/locations` generates a file `spec/requests/api/v1/api_v1_locations_spec.rb`

**Comment by JonRowe** [comment]
Transferred from rspec-core as this is an rspec-rails issue. Theres probably some logic missing from the request spec generator that's in the other ones for anyone who wishes to take a shot at this.

**Comment by hegwin** [comment]
Ah…I just see this PR: https://github.com/rspec/rspec-rails/pull/2057/files

So is it fixed and released?

**Comment by JonRowe** [comment]
Ah yes, I must have the memory of a goldfish, yes this was released as `rspec-rails` 3.8.2, closing as a duplicate, feel free to comment again if you can still replicated this on 3.8.2
